tools/marvin/marvin/config/test_data.py

```diff
@@ -784,6 +784,18 @@
         "format": "OVA",
         "ispublic": "true"
     },
+    "coreos": {
+        "displaytext": "coreos",
+        "name": "coreos",
+        "passwordenabled": False,
+        "ostype": "Coreos",
+        "urlvmware":"http://10.147.28.7/templates/coreos/coreos_production_vmware.ova",
```

Comment:
Thanks @runseb for catching this.
@sanjeevneelarapu the IP is not available for us, please host this file publicly somewhere.